README.md

```diff
@@ -85,6 +85,11 @@ use of standard NumPy/dask arrays as its underlying data storage.
 The documentation for Iris is available at <https://scitools.org.uk/iris/docs/latest>,
 including a user guide, example code, and gallery.
 
+## Work in progress
```

Comment:
Could we remove this title?
To me, it reads as if this section is a WIP in the sense that at some point we hope for it not to be "in progress anymore" and to actually finalise on the contents. But that's not what will happen. It will always be "in progress".

I think it would be fine if 

> A documentation build for the latest development code is visible at https://scitools-docs.github.io/iris/master/index.html.

is part of the section above

Reply:
ok!

Reply:
@pp-mo It looks like this title is still yet to be removed.

Reply:
Sorry, that change got lost when I reverted something.
Will reinstate...